inference-engine/src/transformations/include/transformations/convert_extract_image_patches_to_reorg_yolo.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <ie_api.h>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+    namespace pass {
```

Comment:
You have a code style failure in this file. Remove the indentation for the "namespace pass".

Reply:
Fixed.